docs/install/install-kn.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Installing the `kn` Client"
+linkTitle: "Installing the `kn` Client"
+weight: 10
+type: "docs"
+---
+
+## Install `kn` using the nightly executable binary
```

Comment:
Shouldn't we also offer installing the latest release of the client? I'm not sure if most users want to download and work with the nightly release.

https://github.com/knative/client/blob/master/docs/README.md has instructions on downloading the most recent release.

Reply:
I also think we should offer some information about who should install the nightly executable, and probably a warning that it may not be super stable?

Reply:
The 'latest' release is also covered under the *kn container images* section of the docs: https://knative.dev/development/install/install-kn/

I will check in with the CLI team about what to put for a warning/audience note about the nightly executable.